gui/wxpython/startup/utils.py

```diff
@@ -18,41 +18,48 @@
 
 import os
 import shutil
+import tempfile
+import sys
 
 
 def get_possible_database_path():
-    """Finds a path to what is possibly a GRASS Database. 
+    """Finds the directory to what is possibly a GRASS Database.
 
     Looks for directory named grassdata in the usual locations.
 
-    Returns the path as a string or None if nothing was found, so the
-    return value can be used to test if the directory was found.
+    Returns the path as a string or None if nothing was found, so the return 
+    value can be used to test if the directory was found.  
     """
-    home = os.path.expanduser('~')
+    home = os.path.expanduser('~')           
+    path = None     
     # try some common directories for grassdata
     # grassdata (lowercase) in home for Linux (first choice)
     # Documents and My Documents for Windows
     # potential translations (old Windows and some Linux)
     # but ~ and ~/Documents should cover most of the cases
-    # ordered by preference and then likelihood
+    # ordered by platform preference      
+            
+    # Search independent "grassdata" directories      
     candidates = [
-        os.path.join(home, "grassdata"),
-        os.path.join(home, "Documents", "grassdata"),
-        os.path.join(home, "My Documents", "grassdata"),
+        os.path.join(home),
+        os.path.join(home, "Documents")
     ]
+    
     try:
         # here goes everything which has potential unicode issues
-        candidates.append(os.path.join(home, _("Documents"), "grassdata"))
-        candidates.append(os.path.join(home, _("My Documents"), "grassdata"))
+        candidates.append(os.path.join(home, _("Documents")))
```

Comment:
This adds Documents into the list of candidates twice in case _("Documents") == "Documents". So you could do:

```
tr_documents = os.path.join(home, _("Documents"))
if tr_documents not in candidates:
   candidates.append(tr_documents)
```